service/src/test/java/datawave/microservice/querymetric/QueryMetricSplitBrainMergePolicyTest.java

```diff
@@ -171,14 +179,15 @@ private BaseQueryMetric.PageMetric newPageMetric() {
     }
     
     private Config newConfig(String mergePolicy, String mapName) {
-        Config config = getConfig().setProperty(GroupProperty.MERGE_FIRST_RUN_DELAY_SECONDS.getName(), "5")
-                        .setProperty(GroupProperty.MERGE_NEXT_RUN_DELAY_SECONDS.getName(), "3");
+        Config config = new Config().setProperty(ClusterProperty.MERGE_FIRST_RUN_DELAY_SECONDS.getName(), "5")
+                        .setProperty(ClusterProperty.MERGE_NEXT_RUN_DELAY_SECONDS.getName(), "3");
         
-        config.getGroupConfig().setName(HazelcastUtils.generateRandomString(10));
+        config.setClusterName(HazelcastUtils.generateRandomString(10));
         
```

Comment:
```suggestion
        
        // We don't want the test to rely on multicast. Use ip discovery instead.
        // When omitting the port, Hazelcast will look for members at ports 5701, 5702, etc
        JoinConfig joinConfig = config.getNetworkConfig().getJoin();
        TcpIpConfig tcpIpConfig = joinConfig.getTcpIpConfig();
        joinConfig.getMulticastConfig().setEnabled(false);
        tcpIpConfig.addMember("127.0.0.1");
        tcpIpConfig.setEnabled(true);
```

Reply:
I'm not sure this is going to work with multiple hazelcast-based tests running concurrently.  Part of the reason I made this change was because the hazelcast instance from one test was clustering with one from another test.

Reply:
Do these changes make it work for mac?  I'll try to run it through github CI with these changes to see if everything works.